crates/uv-installer/src/satisfies.rs

```diff
@@ -205,7 +205,9 @@ impl RequirementSatisfaction {
                 }
 
                 // Does the package have dynamic metadata?
-                if is_dynamic(requested_path) {
+                // TODO(charlie): Separate `RequirementSource` into `Path` and `Directory`.
+                if requested_path.is_dir() && is_dynamic(requested_path) {
+                    trace!("Dynamic");
```

Comment:
Better message here?

Reply:
Oh sorry I thought this was already present.